axelor-sale/src/main/resources/views/SaleOrder.xml

```diff
@@ -801,9 +807,21 @@
 	                     	 {statusSelect >= 2 && <Box as="dd" m={0} px={3} py={1}>{$fmt('advanceTotal')}</Box>}
 	                     </Box>
 	                  </>
-	              ]]></viewer>
+                    ]]>
+        </viewer>
       </field>
     </panel>
+    <panel name="globalDiscountPanel" title="Global discount"
+      if="__config__.app.getApp('sale')?.getIsGlobalDiscountEnabledOnSaleOrder()"
+      showIf="clientPartner &amp;&amp; saleOrderLineList &amp;&amp; saleOrderLineList.length != 0"
+      sidebar="true">
+      <field name="discountTypeSelect" onChange="action-sale-order-method-on-line-change"/>
+      <field name="discountAmount" showIf="discountTypeSelect != 0"
+        x-currency="$discountCurrency" x-scale="$discountScale"
```

Comment:
Hello,

`$` is not mandatory anymore for dummy field (replaced with propriety `x-dirty`)